src/dbnode/persist/fs/index_lookup.go

```diff
@@ -41,22 +39,17 @@ type nearestIndexOffsetLookup struct {
 	summaryIDsOffsets []xmsgpack.IndexSummaryToken
 	// bytes from file mmap'd into anonymous region
 	summariesMmap []byte
-	// reusable decoder stream
-	decoderStream  xmsgpack.DecoderStream
-	msgpackDecoder *msgpack.Decoder
-	isClone        bool
+	isClone       bool
 }
 
 func newNearestIndexOffsetLookup(
 	summaryIDsOffsets []xmsgpack.IndexSummaryToken,
 	summariesMmap []byte,
-	decoderStream xmsgpack.DecoderStream,
+	decoderStream xmsgpack.ByteDecoderStream,
```

Comment:
Does this get used anymore?  Maybe don't need to pass it in anymore.

Reply:
yeah it's still used to parse the mmap (to generate the summaries slice) and all that so we still have to pass it in, it's just not retained by the struct anymore.

Reply:
Can we remove the param from this constructor then if it doesn't get set on the struct? (it literally looks like an unused var for this method)

Reply:
done